sherpa/astro/ui/tests/test_astro_ui.py

```diff
@@ -257,20 +241,17 @@ def test_psf_model2d(self):
     # bug #12503
     def test_psf_pars_are_frozen(self):
         ui.load_psf('psf', ui.beta2d.p1)
-        self.assertEqual([], p1.thawedpars)
+        self.assertEqual([], ui.beta2d.p1.thawedpars)
```

Comment:
I think this would be better written as

    p1 = ui.get_model_component("p1")
    self.assertEqual([], p1.thawedpars)

I'm not sure there's much of a difference, but with my version you also check that the "global" parameter mapping has been updated.